samples/nrf5340/smp_svr/README.rst

```diff
@@ -0,0 +1,171 @@
+
+For clarity, here is PM label to slot to ``<image>`` translation table:
+
+    +-----------------------+--------+-------------+---------------------------+
+    | PM label              | Slot   | -n <image>  |         covers            |
+    +=======================+========+=============+===========================+
+    | "mcuboot_secondary"   | slot-1 |     0       | internal application part |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_1" | slot-3 |     1       | networking                |
+    +-----------------------+--------+-------------+---------------------------+
+    | "mcuboot_secondary_2" | slot-5 |     2       | qspi application part     |
+    +-----------------------+--------+-------------+---------------------------+
+
+.. note::
+
+   The ``-e`` option actually means "no erase", and is provided to the mcumgr
+   to prevent it from sending erase command to target, before updating image.
+   The options is always needed when ``-n`` is used for image selection,
+   as the erase command is hardcoded to erase slot-1 ("image-1"),
+   regardless of which slot is uploaded at the time.
+
+Upload the signed image
+=======================
+
+To upload the signed image, use the following command:
+
+.. code-block:: console
+
+   sudo mcumgr <connection string> image upload build/zephyr/<signed_upload.bin> -e -n <image>
+
+   use:
+   `internal_flash_update.bin` file and <image> = 0. for updating the internal application part image.
+   `net_core_app_update.bin` file and <image> = 1. for updating the networking core image
+   `qspi_flash_update.bin` file and <image> = 2. for updating the qspi application part image.
+
+
+.. note::
+
+   <image> values 0 and 2 correspond to the internally and the externally linked parts of application, respectively.
+   These two images need to be uploaded always in pairs as they are linked together.
+
+Test the image
+==============
+
+In order to instruct MCUboot to swap the images we need to test the image first,
+obtain image SHA:
+
+.. code-block:: console
+
+   imgtool verify build/zephyr/<signed_upload.bin>
+
+then make sure it boots using the image digest fetched:
+
+.. code-block:: console
+
+   sudo mcumgr <connection string> image test <hash of image>
+
+The selected mode, in this case test, needs to be applied to all images that constitute the complete application.
+After that MCUBoot will swap these images on the next reset.
+
+.. note::
+
+   The sample supports only MCUBoot overwrite-only mode. In that mode the old image is overwritten by the content of the incoming image during firmware update process.
+   Therefore ther is no diference whether given image is marked to be update by either `test` or `confirm`.
+
+Reset remotely
+==============
+
+We can reset the device remotely and observe (use the console output) how
+MCUboot upgarding the images:
+
+.. code-block:: console
+
+   sudo mcumgr <connection string> reset
```

Comment:
Please see [NCS sample template](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html#references), and check if a "References" or "Dependencies" section is needed. Otherwise, disregard this comment.

Reply:
I've added "Dependencies".